tasties_app/tests/test_recipe.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+from tasties_app.models import Recipe, Category
+from django.utils import timezone
+from django.contrib.auth.models import User
+
+
+@pytest.fixture
+def recipe():
+    """
+        This fixture create the data we need for tests
+        and save it in the data base.
+    """
+    user1 = User.objects.create_user(username='john', password='password')
+
+    test_data = ['Test Recipe1', user1, 'Test Description1', 'Test Directions1', 1, 'test_picture1']
+
+    category1 = Category.objects.create(category_name="1")
+    recipe1 = Recipe(title=test_data[0], author_id=test_data[1], description=test_data[2], directions=test_data[3],
+                     publication_date=timezone.now(), minutes_to_make=test_data[4], recipe_picture=test_data[5])
+
+    category1.save()
+    recipe1.save()
+    recipe1.categories.add(category1)
+    recipe1.save()
+    category1.save()
+    return recipe1
+
+
+class TestRecipeModel:
+    """
+        This class test the recipe model and the method.
+    """
+
+    @pytest.mark.django_db
+    def test_save_recipe(self, recipe):
+        assert recipe in Recipe.objects.all()
+
+    @pytest.mark.django_db
+    def test_field_recipe(self, recipe):
```

Comment:
This method name is confusing, I would recommend "test_recipe_fields". Also, in line 40, I would recomment "recipes" instead of "recipe_set" (it is correct that we're getting a Python Set of Recipes, but "recipe_set" might imply setting a value)